support_for_advertising_use_cases.md

```diff
@@ -529,51 +531,55 @@ This use case is common in sales period. Note that sales period can have very pr
 
 Advertisers want to optimize the look and feel of their ads to the audience and to the environment. This helps to drive more performance and improve their overall brand recognition.
 
-This can be done programmatically using historical aggregated and/or personal user data, and contextual information. 
+This can be done programmatically using historical aggregated and/or personal user data, and contextual information.
 
 ## Coupon Management
 
 As an Advertiser I want to be able to add coupons, or any form of targeted discounts to my creatives on specific period of times, potentially for specific products.
 
+# ML Applications
```

Comment:
I like this suggestion. I agree that this is worth calling out as a larger category of use-cases.

Reply:
Happy to hear that!